framework/db/mssql/QueryBuilder.php

```diff
@@ -250,13 +250,45 @@ public function checkIntegrity($check = true, $schema = '', $table = '')
         return $command;
     }
 
+    protected function buildAddCommentSql($comment, $table, $column = null)
+    {
+        $tableSchema = $this->db->schema->getTableSchema($table);
```

Comment:
Why can’t we just use `Schema::defaultSchema` here?

Reply:
Because the point here is not the schema, but to get the table name *without* schema (line 258). And since I had the table schema I took advantage to use it for schema too.